dist/jdataview.js

```diff
@@ -1,23 +1,23 @@
-(function (factory) {
-	var global = this;
-
-	if (NODE || typeof exports === 'object') {
-		module.exports = factory(global);
-	} else
-	if (BROWSER) {
-		if (typeof define === 'function' && define.amd) {
-			define([], function () {
-				return factory(global);
-			});
-		}
-		else {
-			global.jDataView = factory(global);
-		}
-	}
-}(function (global) {
-
-'use strict';
-
+(function (factory) {
```

Comment:
Sure. I reverted the files in `dist` and pushed a squashed commit of all the changes to my fork.

FWIW, it's a bit unusual to include generated files in the git repo.